tidb-lightning/import-into-vs-tidb-lightning.md

```diff
@@ -0,0 +1,138 @@
+
+If you want to continue importing data later, you will need to keep the TiDB Lightning server and the disks on the server for the next import. The cost of using preallocated disks is relatively high, compared with `IMPORT INTO` using Amazon S3 on a pay-as-you-go basis.
+
+### Performance
+
+Currently, there are no performance test comparison results under equivalent test environments between `IMPORT INTO` and TiDB Lightning.
+
+The `IMPORT INTO` performance test results with Global Sort are as follows:
+
+Data sets:
+
+- 40 TiB of data (2.26 billion rows, single row size 19 KiB)
+- 10 TiB of data (565 million rows, single row size 19 KiB)
+
+AWS:
+
+- Importing 40 TiB of data, 10 TiDB (16C32G) 20 TiKV nodes (16C27G), the average single TiDB node import speed is approximately 222 GiB/h.
+- Importing 10 TiB of data, 5 TiDB (16C32G) 10 TiKV (16C27G), the average single TiDB node import speed is approximately 307 GiB/h.
+
+### High availability
+
+#### `IMPORT INTO`
+
+After a TiDB node failure, tasks on that node are automatically transferred to the remaining TiDB nodes to continue running.
+
+#### TiDB Lightning
+
+After a TiDB Lightning instance node failure, you need to perform manual recovery of tasks on a new node based on previously recorded checkpoints.
+
+### Scalability
+
+#### `IMPORT INTO`
+
+Due to the use of Global Sort, data imported into TiKV does not overlap, resulting in better scalability compared with TiDB Lightning.
+
+#### TiDB Lightning
+
+Due to only supporting local sort, data imported into TiKV might overlap when adding TiDB Lightning instances, resulting in more compaction operations for TiKV and limiting scalability relative to `IMPORT INTO`.
+
+## Functionalities not supported by `IMPORT INTO`
+
+Currently, `IMPORT INTO` still lacks some functionalities and cannot fully replace TiDB Lightning in some scenarios, such as:
+
+- Logical import
+
+    Before importing data with `IMPORT INTO`, the target table must be empty. If you need to import data into a table that already contains data, it is recommended to use methods like [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) or direct insertion. TiDB v8.0 supports [bulk DML](/system-variables.md#tidb_dml_type-new-in-v800) for executing large transactions.
+
+- Conflict data handling
+
+    `IMPORT INTO` currently does not support conflict data handling. You need to define the table structure properly to ensure that the data to be imported does not conflict with primary keys (PK) or unique keys (UK). Otherwise, it might cause task failures.
```

Comment:
```suggestion
    `IMPORT INTO` currently does not support conflict data handling. Before the data import, you need to define the table schema properly to ensure that the data to be imported does not conflict with primary keys (PK) or unique keys (UK). Otherwise, it might cause task failures.
```